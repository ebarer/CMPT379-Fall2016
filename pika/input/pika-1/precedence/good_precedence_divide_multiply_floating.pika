exec {

  print 8.5 / 1.7 * 2.1, _n_.
  print 4.7 / 7.4 * 9.9, _n_.
  print 4.7 / 4.5 * 9.5, _n_.
  print 5.3 / 7.2 * 1.4, _n_.
  print 2.9 / 1.5 * 5.2, _n_.
  print 4.8 / 8.4 * 3.9, _n_.
  print 3.4 / 9.9 * 3.6, _n_.
  print 7.3 / 6.2 * 9.8, _n_.
  print 4.4 / 6.2 * 2.6, _n_.
  print 2.2 / 5.0 * 2.6, _n_.
  print 1.4 / 4.1 * 8.2, _n_.
  print 4.0 / 6.4 * 8.3, _n_.

  print 4.8 * 7.7 / 8.2, _n_.
  print 2.4 * 6.5 / 7.9, _n_.
  print 7.7 * 6.5 / 5.9, _n_.
  print 6.3 * 1.1 / 5.2, _n_.
  print 7.9 * 1.1 / 1.3, _n_.
  print 7.3 * 4.0 / 5.4, _n_.
  print 8.4 * 2.5 / 4.8, _n_.
  print 8.5 * 1.2 / 2.5, _n_.
  print 9.1 * 8.2 / 2.5, _n_.
  print 3.5 * 1.1 / 6.0, _n_.
  print 4.0 * 8.9 / 7.8, _n_.
  print 6.4 * 4.0 / 4.0, _n_.

  print 7.2 / 6.5 * 3.3 / 6.2, _n_.
  print 7.0 / 2.6 * 4.2 / 5.9, _n_.
  print 8.1 / 7.2 * 2.2 / 2.7, _n_.
  print 1.0 / 4.7 * 9.9 / 7.1, _n_.
  print 7.8 / 7.1 * 3.0 / 8.4, _n_.
  print 3.9 / 4.0 * 6.7 / 5.4, _n_.
  print 6.4 / 4.8 * 2.1 / 9.9, _n_.
  print 3.3 / 7.0 * 1.1 / 7.1, _n_.
  print 4.4 / 7.4 * 2.8 / 6.8, _n_.
  print 3.8 / 7.9 * 8.8 / 6.7, _n_.
  print 2.2 / 6.7 * 3.9 / 8.3, _n_.
  print 6.5 / 6.2 * 3.0 / 2.2, _n_.

  print 9.6 * 5.6 / 7.4 * 6.3, _n_.
  print 3.3 * 7.7 / 5.8 * 5.4, _n_.
  print 7.8 * 7.8 / 8.8 * 7.2, _n_.
  print 1.2 * 4.2 / 4.3 * 2.5, _n_.
  print 5.1 * 8.8 / 4.2 * 1.5, _n_.
  print 4.8 * 1.9 / 9.0 * 2.0, _n_.
  print 9.8 * 3.0 / 6.9 * 3.6, _n_.
  print 8.0 * 2.7 / 6.9 * 9.0, _n_.
  print 1.0 * 5.6 / 9.8 * 4.3, _n_.
  print 6.2 * 3.5 / 8.6 * 4.7, _n_.
  print 5.1 * 8.6 / 8.3 * 6.2, _n_.
  print 4.2 * 5.6 / 6.0 * 7.6, _n_.

}

