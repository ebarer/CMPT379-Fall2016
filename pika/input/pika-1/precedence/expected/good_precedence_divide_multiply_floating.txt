10.5
6.28784
9.92222
1.03056
10.0533
2.22857
1.23636
11.5387
1.84516
1.144
2.8
5.1875
4.50732
1.97468
8.48305
1.33269
6.68462
5.40741
4.375
4.08
29.848
0.641667
4.5641
6.4
0.589578
1.91656
0.916667
0.296674
0.392354
1.20972
0.282828
0.0730382
0.244833
0.631778
0.154289
1.42962
45.7686
23.6576
49.7782
2.93023
16.0286
2.02667
15.3391
28.1739
2.45714
11.8593
32.7629
29.792
